Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM node:13.8.0-alpine3.10
+FROM node:12.16.2-alpine3.9
```

Comment:
was there a specific reason we moved to this one?

Reply:
Yes. Node.js version 13.x.x will become unsupported soon (as all odd numbers versions). Meanwhile, version 12.x.x is widely used and will be supported until October 2021